lib/rubocop/cop/rspec/describe_class.rb

```diff
@@ -48,12 +52,31 @@ def on_top_level_describe(node, args)
           return if shared_group?(root_node)
           return if valid_describe?(node)
 
+          if first_argument_empty?(node)
+            return add_offense(node, location: :expression,
+                                     message: FIRST_ARGUMENT_REQUIRED_MSG)
+          end
+
           describe_with_metadata(node) do |pairs|
             return if pairs.any?(&method(:rails_metadata?))
           end
 
           add_offense(args.first, location: :expression)
         end
+
+        private
+
+        def valid_describe?(node)
+          if first_argument_empty?(node) && !first_argument_required?
+            return true
+          end
+
+          describe_with_const?(node)
+        end
```

Comment:
This method reads a bit odd to me… Would something like the following work?
```
return false if first_argument_required? && node.arguments.none?

describe_with_const?
```

Reply:
Had the same feeling, but did not know how to solve it. Now its better.